src/com/facebook/buck/android/UnzipAar.java

```diff
@@ -203,7 +213,7 @@ public BuildOutput initializeFromDisk(OnDiskBuildInfo onDiskBuildInfo) throws IO
   @Override
   @Nullable
   public Path getPathToOutput() {
-    return null;
+    return copiedAar;
```

Comment:
It seems odd that we're listing essentially a copy of the input file as the output.  Shouldn't the output of `UnzipAar` be `unpackDirectory`?